views/events/calls_events.view.lkml

```diff
@@ -0,0 +1,217 @@
+# The name of this view in Looker is "Calls Events"
+view: calls_events {
+  # The sql_table_name parameter indicates the underlying database table
+  # to be used for all fields in this view.
+  sql_table_name: "EVENTS"."CALLS_EVENTS"
+    ;;
+  drill_fields: [id]
+  # This primary key is the unique key for this table in the underlying database.
+  # You need to define a primary key in a view in order to join to other views.
```

Comment:
these look to be generic comments (are they coming from the template?), should we remove them?